addons/mail/static/tests/helpers/model_definitions_setup.js

```diff
@@ -106,6 +110,6 @@ insertRecords('res.users', [
 ]);
 insertRecords('res.partner', [
     { active: false, display_name: "Public user", id: TEST_USER_IDS.publicPartnerId, },
-    { display_name: "Your Company, Mitchell Admin", id: TEST_USER_IDS.currentPartnerId, name: "Mitchell Admin", },
+    { display_name: "Your Company, Mitchell Admin", id: TEST_USER_IDS.currentPartnerId, name: "Mitchell Admin", user_ids: [TEST_USER_IDS.currentUserId], },
```

Comment:
Are you sure this is necessary in master? Inverse are now computed automatically, so writing `partner_id` on user (as done above) should be enough.